packages/react-storage/src/components/StorageImage/StorageImage.tsx

```diff
@@ -1,53 +1,23 @@
-import * as React from 'react';
```

Comment:
I'm not sure we can remove this, I think it's needed by React 16? https://legacy.reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html#removing-unused-react-imports

Will let @calebpollman confirm.